packages/editor/src/dataviews/store/private-actions.ts

```diff
@@ -164,15 +174,18 @@ export const registerPostTypeSchema =
 			postTypeConfig.supports?.thumbnail &&
 				currentTheme?.[ 'theme-supports' ]?.[ 'post-thumbnails' ] &&
 				featuredImageField,
-			titleField,
+			! isDeisgnPostType &&
```

Comment:
Aren't we missing the pattern post type check here? It's hidden because we're not using `usePostFields` in the page-patterns & page-templates, but rather importing the fields directly.

This logic can be simpler by making it specific:

```js
postTypeConfig.supports?.title && postType === 'page' && pageTitleField
postTypeConfig.supports?.title && postType === 'post' && titleField
postTypeConfig.supports?.title && postType === 'wp_block' && patternTitleField
postTypeConfig.supports?.title && postType === 'wp_template' && templateTitleField
```

Reply:
I updated [here](https://github.com/WordPress/gutenberg/pull/67449/commits/77055d20e8af0987d584cf53289caad2e9c88c10). I just didn't add the `post` check because `titleField` is fine for any post type.